src/components/attendee/Attendance/Attendance.tsx

```diff
@@ -12,11 +12,15 @@ type AttendanceOptions = {
 };
 
 export const Attendance: React.FC<AttendanceOptions> = ({
-  totalUsersCount = 0,
+  totalUsersCount,
   usersSample,
 }) => {
   const containerRef = useRef(null);
 
+  if (!totalUsersCount) {
```

Comment:
Maybe it's OOS, but this prop should be required instead of optional. The complier should 'cry' and force us to pass it or maybe both props. WDYT?

Right now I can render this component like this, and it won't render anything in the DOM:

```tsx
<Attendance/>
```

Reply:
Yes, I think that makes sense. Mostly wasn't sure if there are some non-obvious reasons for it to be optional. Will change it quickly

Reply:
Sounds good. A lot of components need this change, maybe it's best if we do it in a separate PR.